### ISSUE TITLE
importchannel command fails in postgresql when channels had strings in non utf-8 format

### ISSUE THREAD

**Issue by jredrejo** [learningequality/kolibri#11780]
## Observed behavior
When importing some channels into a kolibri running over Postgresql, tags data is converted to bytes, having a length that exceeds the 30 chars max width of the tags column.

For example
`'formación de imágenes'`
is converted by the code into
`b'formaci\xc3\xb3n de im\xc3\xa1genes'`
which exceeds 30 chars

### Possible solutions:
One possible solution for this would be to cut the string to the column limit when it's byte encoded, using something like
`value.encode(encoding)[:column_size].decode(encoding, 'ignore')`

Another solution could be replacing current code using `execute_values` that forces the byte conversion by a sql query passed to` cursor.execute` including 
```
 ON CONFLICT (id) DO 
UPDATE 
SET 
  tag_name = COALESCE(
    EXCLUDED.tag_name, SOURCE.tag_name
  );
```
to ensure updates

## Errors and logs

```
$ kolibri manage importchannel network --baseurl=https://studio.learningequality.org 07cd1633691b4473b6fda08caf826253
INFO     2024-01-24 19:26:09,932 Running Kolibri with the following settings: kolibri_data_portal.settings.dev
INFO     2024-01-24 19:26:10,082 Invoking command importchannel network --baseurl=https://studio.learningequality.org 07cd1633691b4473b6fda08caf826253
System check identified some issues:

WARNINGS:
device.UserSyncStatus.user: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
INFO     2024-01-24 19:26:11,575 Downloading data for channel id 07cd1633691b4473b6fda08caf826253
INFO     2024-01-24 19:26:12,807 Importing LocalFile data
INFO     2024-01-24 19:26:12,831 Importing Language data
INFO     2024-01-24 19:26:12,834 Importing ContentNode data
INFO     2024-01-24 19:26:12,960 Importing ChannelMetadata data
INFO     2024-01-24 19:26:12,965 Importing File data
INFO     2024-01-24 19:26:12,982 Importing AssessmentMetaData data
INFO     2024-01-24 19:26:12,985 Importing ContentNode_related data
INFO     2024-01-24 19:26:12,988 Importing ContentNode_has_prerequisite data
INFO     2024-01-24 19:26:12,990 Importing ContentTag data
Traceback (most recent call last):
  File "/datos/le/mio/kolibri-data-portal/venv/bin/kolibri", line 8, in <module>
    sys.exit(main())
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/utils/cli.py", line 200, in invoke
    return super(KolibriDjangoCommand, self).invoke(ctx)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/utils/cli.py", line 373, in manage
    execute_from_command_line(["kolibri manage"] + ctx.args)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/dist/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/tasks/management/commands/base.py", line 28, in handle
    return self.handle_async(*args, **options)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/management/commands/importchannel.py", line 269, in handle_async
    self.download_channel(
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/management/commands/importchannel.py", line 107, in download_channel
    self._transfer(
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/management/commands/importchannel.py", line 170, in _transfer
    self._start_file_transfer(
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/management/commands/importchannel.py", line 240, in _start_file_transfer
    import_ran = import_channel_by_id(
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/management/commands/importchannel.py", line 27, in import_channel_by_id
    return channel_import.import_channel_from_local_db(
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/utils/channel_import.py", line 1221, in import_channel_from_local_db
    return import_manager.run_and_annotate()
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/utils/channel_import.py", line 967, in run_and_annotate
    import_ran = self.import_channel_data()
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/utils/channel_import.py", line 931, in import_channel_data
    self.table_import(model, row_mapper, table_mapper)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/utils/channel_import.py", line 721, in table_import
    result = self.postgres_table_import(model, row_mapper, table_mapper)
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/kolibri/core/content/utils/channel_import.py", line 655, in postgres_table_import
    execute_values(
  File "/datos/le/mio/kolibri-data-portal/venv/lib/python3.10/site-packages/psycopg2/extras.py", line 1292, in execute_values
    cur.execute(b''.join(parts))
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(30)
```

## Expected behavior
Importing a channel into a postgresql db should work, no matter the channel language

## User-facing consequences
Some channels can't be imported

## Steps to reproduce
Run the importchannel command on a kolibri installation running on postgresql

**Comment by jredrejo** [comment]
Postponing this until psycopg is updated after django 3.x migration

**Comment by rtibbles** [comment]
Fixed in #12709
